src/Parser/EnhancementShaman/Modules/ShamanCore/AlphaWolf.js

```diff
@@ -0,0 +1,41 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Module from 'Parser/Core/Module';
+
+class AlphaWolf extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  feralSpiritCastCount = 0;
+  feralSpiritCastFlag = false;
+  missedAlphaWolfCount = 0;
+
+  on_byPlayer_cast(event) {
+    if(this.feralSpiritCastFlag) {
+      if(event.ability.guid !== SPELLS.CRASH_LIGHTNING.id) {
+        this.missedAlphaWolfCount += 1;
+      }
+      this.feralSpiritCastFlag = false;
+    }
+
+    if(event.ability.guid === SPELLS.FERAL_SPIRIT.id) {
+      this.feralSpiritCastFlag = true;
+      this.feralSpiritCastCount += 1;
+    }
+  }
+
+  suggestions(when) {
+    when(this.missedAlphaWolfCount).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('Make sure Crash Lightning is always cast directly after Feral Spirits to activate Alpha Wolf, even on single target')
```

Comment:
Please include what was observed. E.g. You didn't always casts Chain Lightning immediately after Feral Spirits. 

Also please turn the spells into SpellLinks.